core/drivers/crypto/caam/hal/common/hal_cfg.c

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: BSD-2-Clause
 /*
- * Copyright 2017-2019 NXP
+ * Copyright 2017-2019, 2021 NXP
```

Comment:
s/2017-2019, 2021/2017-2021/

Reply:
Here in NXP we follows this approach, but if OP-TEE follows this approach you mentioned, I will change that.

Reply:
As you wish.

Reply:
Kept it NXP style.